int32 var
---
sensor_msgs/PointCloud output_cloud
